index.html

```diff
@@ -74,31 +74,61 @@ <h2>I'm a Full Stack Developer</h2>
+                <span class="project-bar two"></span>
+            </button>
+
+            <div class="preview-project-content">
+                
+                <h1 class="preview-project-title">Daily Selection</h1>
+                <ul class="preview-tags tags project-content div-flex">
+                   
+                </ul>
+                <div class="main-preview-project div-flex">
+                    <img class="preview-project-img" src="vectors/Img_Placeholder2.png" alt="preview-project">
+                    <div class="preview-card-description div-flex">
+                        <p class="mobile-preview-card-p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+                        <p class="preview-card-p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.</p>
+                    <div class="preview-container-btns">
+                        <a class="btn preview-project-btn preview-see-live" href="" target="_blank" rel="noopener">See-live <i class="fa-solid fa-arrow-up-right-from-square"></i></a>
+                        <a class="btn preview-project-btn preview-see-source" href="" target="_blank" rel="noopener">See-source <i class="fa-brands fa-github-alt"></i></a>
+                    </div>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+
+<!--HERE ENDS PREVIEW PROJECT SECTION-->
+
         <section class="works div-flex">
             <div class="works-title-container div-flex">
                 <h2 class="works-title">My Recent Works</h2>
                 <hr class="works-divider div-flex">
             </div>
             <div class="daily-selection div-flex">
-                <a class="img-placeholder-mobile div-flex" href=""><img class="img-daily"
-                        src="vectors/Img_Placeholder2.png" alt="Project Image mobile"></a>
-                <a class="img-placeholder-desktop div-flex" href=""><img class="img-daily-desktop"
+                <a class="preview-btn img-placeholder-mobile div-flex"><img class="img-daily"
+                        src="vectors/Img_Placeholder2.png" alt="preview Image mobile"></a>
+                <a class="preview-btn img-placeholder-desktop div-flex"><img class="img-daily-desktop"
```

Comment:
- [ ] As requested in the project requirements, The projects section should be created dynamically using Javascript. all already created HTML code for the projects section list should be removed, and instead, it should be added via JavaScript. You should only have a container for the section in the HTML file.
- [ ] Create an array of objects to store JavaScript objects containing the information for each project. e.g
```javascript
const projects = [{
    name: "Project title",
    description: "projects description",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
You can then iterate through the project array to populate the page and the pop-ups. this will make adding more projects in the future easy.

- [ ] Also on the desktop version, each project card should have a button to view the project in the pop-ups. You can make it that when the card is hovered, the button shows.

Reply:
Could you be more clear? When I open the pop up all the project information stored in the objects that I created is replaced in his corresponding site in time. It is necessary to do the same thing in each project card on the grid??

And the button in each project is not a requirement in the template of Figma and it is not a requirement for this project, you can click the card, and the project pop up and the project is going to appear. What do you mean by "project section"? are you talking about the project grid or the pop-up?

On the other hand, my pop-up is dynamic and is using the information stored in the object for each project in javascript, I used an Object constructor to store all the information of each project.

Reply:
@carreraprogrammer, yes, in the HTML file, there should be only the container for all the projects. both the project cards and the pop-ups should be generated dynamically.
In your js file, you should have an array holding all the projects like the example I gave at the top. In the example, I have only one object, but yours should have objects for the number of projects card.
Then you iterate through the array to dynamically create the project card, and the pop-ups also, using the details you've provided in the objects.
If it's still not clear, you can contact me on slack, and I can explain it better over a call.